src/main/scala/nl/knaw/dans/easy/bagindex/command/CommandLineOptions.scala

```diff
@@ -59,26 +59,20 @@ class CommandLineOptions(args: Array[String], properties: PropertiesConfiguratio
   footer("")
 
   object interaction {
-    /**
-     * Interactive command line interface to ask whether the user is sure to index the whole bag-store
-     * and thereby delete all current data in the bag-index.
-     *
-     * @return `true` if the user wants to proceed, `false` otherwise
-     */
     def deleteBeforeIndexing(): Boolean = {
       @tailrec
       def recursiveAsk(): Boolean = {
+        print("Proceed? (y/n): ")
         StdIn.readLine().toLowerCase match {
-          case "yes" => true
-          case "no" => false
+          case "y" => true
+          case "n" => false
           case _ =>
-            println("either use 'Yes' or 'No'")
+            println("Answer y or n.")
             recursiveAsk()
         }
       }
 
-      println("Before the bag-store is indexed, the current index needs to be deleted?")
-      println("Do you want to proceed? (Yes/No)")
+      println("WARNING: you are about to rebuild the entire index. You should NOT attempt this while the daemon is running.")
```

Comment:
* The deposit will fail. I agree that the clients of this service should have a retry policy. This is the more general point of making our services more robust. That shouldn't prevent us from selecting the lesser of two evils in the meantime, which, in my opinion, is letting the deposit fail (vs. potentially let a duplicate deposit happen for instance).

* If everything is inside one transaction, you may be right. In that case the service could stay up for the whole time and the interaction could be removed entirely. But I am not sure how long it would block the whole database if the bag-stores are large, so it may in fact amount to the same thing as the service shutting down for a while.